packages/forma-36-react-components/src/components/EmptyState/EmptyState.css

```diff
@@ -0,0 +1,29 @@
+@import 'settings/colors';
+@import 'settings/typography';
+@import 'settings/transitions';
```

Comment:
We can remove the color and transition imports as we won't be using them.

Please also add `@import 'settings/dimensions';` so that we can use our f36 spacing variables